civis/ml/tests/test_model.py

```diff
@@ -123,10 +127,11 @@ def foo(arg):
 
 @mock.patch.object(_model.cio, 'file_to_civis', return_value=-11)
 def test_stash_local_data_from_file(mock_file):
-    with tempfile.TemporaryDirectory() as tempdir:
-        fname = os.path.join(tempdir, 'filename')
+    with tempfile.NamedTemporaryFile() as tempfname:
```

Comment:
It looks like a `NamedTemporaryFile` here won't work on Windows: 

"Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later)." (https://docs.python.org/2/library/tempfile.html?highlight=tempfile#tempfile.NamedTemporaryFile)

Perhaps that doesn't matter in the tests?